### PR TITLE
Testing semantic

### LINKED ISSUE #7: Testing semantic code
testing all semantic logic already in master, TypeCollector, TypeBuilder, TypeChecker...

### PR DESCRIPTION
**Pull request by DalyPerez**
closes #7

### CODE REVIEW

**Review by ginrod — CHANGES_REQUESTED** [review]
removing outdate comments and testingprints

File: src/semantic_analyzer.py [hunk 1]
```diff
@@ -56,8 +57,10 @@ def visit(self, node):
         try:
             self.current_type = self.context.get_type(node.name)
             if node.parent:
+                #TODO: falla xq no existen aun los tipos builtin
```
Comment (ginrod) [inline src/semantic_analyzer.py]: remove this

File: src/semantic_analyzer.py [hunk 2]
```diff
@@ -74,6 +77,7 @@ def visit(self, node):
             return_type = self.context.get_type(node.return_type)
             self.current_type.define_method(
                 node.name, param_names, param_types, return_type)
+            print(self.context.get_type(node.name))
```
Comment (ginrod) [inline src/semantic_analyzer.py]: remove this

File: src/semantic_analyzer.py [hunk 3]
```diff
@@ -105,7 +109,7 @@ def visit(self, node, scope=None):
 
     @visitor.when(AST.Class)
     def visit(self, node, scope):
-        self.current_type = self.context.get_type(node.id)
+        self.current_type = self.context.get_type(node.name)  #TODO: creo que aqui debe ser node.name
```
Comment (ginrod) [inline src/semantic_analyzer.py]: remove comment

File: src/semantic_analyzer.py [hunk 4]
```diff
@@ -240,11 +244,14 @@ def analyze(self):
         collector = TypeCollector(self.errors)
         collector.visit(self.ast)
         context = collector.context
-        print(context)
-
-        #'=============== BUILDING TYPES ================'
+        
+        print("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!")
```
Comment (ginrod) [inline src/semantic_analyzer.py]: remove this

File: src/semantic_analyzer.py [hunk 5]
```diff
@@ -240,11 +244,14 @@ def analyze(self):
...
+        # #'=============== BUILDING TYPES ================'
         builder = TypeBuilder(context, self.errors)
         builder.visit(self.ast)
+        print(context)
```
Comment (ginrod) [inline src/semantic_analyzer.py]: remove this

**Review by ginrod — CHANGES_REQUESTED** [review]
remove prints and comments

**Review by dayanyalfaro — APPROVED** [review]

**Review by ginrod — APPROVED** [review]

**Pull request merged**